src/eva/data/mon_data_space.py

```diff
@@ -0,0 +1,280 @@
+# (C) Copyright 2021-2023 NOAA/NWS/EMC
+#
+# (C) Copyright 2021-2023 United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration. All Rights Reserved.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+
+# --------------------------------------------------------------------------------------------------
+
+
+import os
+import numpy as np
+
+from xarray import Dataset, concat
+from scipy.io import FortranFile
+from datetime import datetime
+
+from eva.eva_base import EvaBase
+from eva.utilities.config import get
+from eva.utilities.utils import parse_channel_list
+
+# --------------------------------------------------------------------------------------------------
+
+
+class MonDataSpace(EvaBase):
+
+    # ----------------------------------------------------------------------------------------------
+
+    def execute(self, data_collections, timing):
+
+        # Loop over the datasets
+        # ----------------------
+        for dataset in self.config.get('datasets'):
+
+
+            # Set the collection name
+            # -----------------------
+            collection_name = get(dataset, self.logger, 'name')
+
+            # Get control file and parse
+            # --------------------------
+            control_file = get(dataset, self.logger, 'control_file')
```

Comment:
That makes sense.  Force of habit in the yaml file.

Note that I figured out that the control file was in the wrong location in the yaml file.  It was initially an element (if that's the right term) of data instead of dataset.  I've corrected that.